db/mockdata.py

```diff
@@ -0,0 +1,183 @@
+"""
+This module contains classes required by the compact API.
```

Comment:
Also, there are multiple classes in one python module. It would be nice to separate those out into different files. Maybe start naming the whole set of files lastfm_session.py, lastfm_token.py ... ?